packages/ibm-products/src/global/js/hooks/useOverflowItems/useOverflowItems.mdx

```diff
@@ -0,0 +1,73 @@
+import { Story, Canvas } from '@storybook/addon-docs';
+import * as UseOverflowItemsStories from './useOverflowItems.stories';
+import { useOverflowItems } from '.';
+
+# useOverflowItems
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Hook API](#hook-api)
+- [Example usage](#example-usage)
+- [Example implementation](#example-implementation)
+
+## Overview
+
+The `useOverflowItems` hook can be used for handling overflow behavior of multiple DOM elements inside of a given area.
+
+For example, if you need to display a certain number of `Tag` elements in a container and render additional values in a popover, you could use this hook to do just that.
+
+## Hook API
+
+```jsx
+// Hook arguments
+items: T[] = [], // An iterable list of items
```

Comment:
should we be more clear with the type `T`?

Reply:
not sure if it's a generic type?

Reply:
If it is maybe just stating that T is the generic type passed in so that way non-tyypescript savvy people can understand a bit more

Reply:
yeah it's a generic type. i can update it to make it more clear.